src/Components/PhotosWall/PhotosWall.styles.js

```diff
@@ -1,25 +1,74 @@
 import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const PhotosContainer = styled.div`
+	overflow: hidden;
+`;
 
 export const ImageContainer = styled.div`
+	position: relative;
 	margin: 10px;
+	background-color: ${(props) => props.background};
+	overflow: hidden;
+	cursor: pointer;
+`;
+
+export const StyledLink = styled(Link)`
+	cursor: pointer;
+	color: black;
+	text-decoration: none;
+	&.profile_pic {
+		padding: 5px;
+		display: flex;
+		align-items: center;
+		gap: 5px;
+	}
```

Comment:
the point of styled components is to not use classes

https://styled-components.com/docs/advanced#referring-to-other-components